src/pages/low/hardware_topics/electronic-design.md

```diff
@@ -19,6 +19,6 @@ Luos' One-wire official connector is: <a href="https://octopart.com/df11-4dp-2ds
 Luos' RS485 official connector is: <a href="https://octopart.com/df11-8dp-2ds%2824%29-hirose-39521447" target="_blank">*DF11-8DP-2DS*</a>.
 
 ## Compatible MCUs
-Luos manages any type of microcontrollers, but they need to be added manually to the library. If your microcontroller is not supported yet, please contact us:
+Luos can manage any type of microcontrollers as long as they are first added to the library manually. If your microcontroller is not supported yet, please contact us:
```

Comment:
```suggestion
Luos can manage any type of microcontrollers as long as they are first added to the library manually. If your microcontroller is not yet supported, please contact us:
```

Reply:
@jlangbridge I do agree this flows better 👍  Feel free to commit the suggestion if this is what you meant 👍